test/cbmc/proofs/Makefile-project-defines

```diff
@@ -0,0 +1,38 @@
+# -*- mode: makefile -*-
+# The first line sets the emacs major mode to Makefile
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+################################################################
+# Use this file to give project-specific definitions of the command
+# line arguments to pass to CBMC tools like goto-cc to build the goto
+# binaries and cbmc to do the property and coverage checking.
+#
+# Use this file to override most default definitions of variables in
+# Makefile.common.
+################################################################
+
+# Flags to pass to goto-cc for compilation (typically those passed to gcc -c)
+COMPILE_FLAGS = -fPIC
+
+# Flags to pass to goto-cc for linking (typically those passed to gcc)
+LINK_FLAGS = -std=gnu90
+
+# Preprocessor include paths -I...
+# Consider adding
+#     INCLUDES += -I$(CBMC_ROOT)/include
+# You will want to decide what order that comes in relative to the other
+# include directories in your project.
+#
+INCLUDES += -I$(SRCDIR)/source/include
+INCLUDES += -I$(SRCDIR)/test/unit-test
```

Comment:
Are these paths correct? What is SRCDIR being set to? Is this the root of the directory? I'm wondering why we would include the unit-test directory. It seems like this would not be relevant.

Reply:
Yes, the SRCDIR is set to the root of the directory. I'm not sure why the unit-test was included too. I will remove the include line if it is not required.

Reply:
There are some header files which are used by the ota library present in the unit-test directory.

Reply:
What files are these? If they are things like the ota_config.h header file, then we should have a unique copy of this file for CBMC proofs. This config file is only needed though if the default config files are not sufficient.

Reply:
Yes it's the config file.